src/custom/pages/App/index.tsx

```diff
@@ -63,8 +65,8 @@ export const BodyWrapper = styled.div`
     align-items: flex-start;
   `}
 
-  ${({ theme }) => theme.mediaWidth.upToMedium`
-    padding: 0;
+  ${({ theme, location }) => theme.mediaWidth.upToMedium`
+    padding: ${location.pathname === '/swap' ? '0 0 16px' : '0 16px 16px'};
```

Comment:
Use the new enum here?

```suggestion
    padding: ${location.pathname === Routes.SWAP ? '0 0 16px' : '0 16px 16px'};
```

Reply:
Yes!